README.md

```diff
@@ -1,5 +1,7 @@
 # Vision Transformer and MLP-Mixer Architectures
 
+**Update (2.7.2021)**: Added the "When Vision Transformers Outperform ResNets..." paper, and SAM optimized ViT and MLP-Mixer checkpoints.
```

Comment:
Could you break lines at 80 characters?
(We only have lines with links & tables running over the character limit)